docs/root/configuration/http_filters/grpc_http1_reverse_bridge_filter.rst

```diff
@@ -0,0 +1,35 @@
+.. _config_http_filters_grpc_reverse_bridge:
+
+gRPC HTTP/1.1 reverse bridge
+============================
+
+* gRPC :ref:`architecture overview <arch_overview_grpc>`
+* :ref:`v2 API reference <envoy_api_field_config.filter.network.http_connection_manager.v2.HttpFilter.name>`
+* This filter should be configured with the name *envoy.grpc_http1_reverse_bridge*.
+
+This is a filter that enables converting an incoming gRPC request into a HTTP/1.1 request to allow a server
+that does not understand HTTP/2 or gRPC semantics to handle the request.
+
+The filter works by:
+
+* Checking the content type of the incoming request. If it's a gRPC request, the filter is enabled.
+* The content type is modified to a configurable value. This can be a noop by configuring ``application/grpc``.
+* The gRPC frame header is optionally stripped from the request body. The content length header will be adjusted if so.
+* On receiving a response, the content type of the response is validated and the status code is mapped to a grpc-status
+  which is inserted into the response trailers.
+* The response body is optionally prefixed by the gRPC frame header, again adjusting the content length header if necessary.
+
+Due to being mapped to HTTP/1.1, this filter will only work with unary gRPC calls.
+
+gRPC frame header management
+----------------------------
+
+By setting the withhold_grpc_frame option, the filer will assume that the upstream does not understand any gRPC semantics
```

Comment:
typo "filer"

super nit: as long as you are in here again can you try to flow doc lines to ~100 col if possible? Same in other files.